apps/frontend/src/components/Sidebar.tsx

```diff
@@ -180,181 +127,70 @@ const Sidebar: React.FC = () => {
-          onClick={() => showModal(<FeedbackModal />)}
-          onKeyDown={(e) => {
-            if (e.key === "Enter" || e.key === " ") {
-              showModal(<FeedbackModal />)
-            }
-          }}
-        >
-          <Icon
-            icon="fluent:question-circle-20-regular"
-            style={{ fontSize: "28px " }}
-          />
-        </div>
-        </>
-      )}
-      </motion.div>
-        {/* 
-        <SidebarLink
-          href="/settings/"
-          icon={
-            <svg
-              width="22"
-              height="22"
-              viewBox="0 0 22 22"
-              fill="none"
-              xmlns="http://www.w3.org/2000/svg"
-            >
-              <path
-                d="M20.0668 6.20845L19.5858 5.39468C19.2221 4.77924 19.0403 4.47152 18.7308 4.34881C18.4214 4.22611 18.0714 4.32293 17.3715 4.51658L16.1826 4.8431C15.7358 4.94357 15.2669 4.88657 14.8589 4.68218L14.5307 4.49752C14.1808 4.27903 13.9117 3.95689 13.7627 3.57823L13.4374 2.63072C13.2234 2.0037 13.1164 1.69019 12.8617 1.51086C12.6071 1.33154 12.2688 1.33154 11.5922 1.33154H10.506C9.82951 1.33154 9.49121 1.33154 9.23651 1.51086C8.98187 1.69019 8.87489 2.0037 8.66095 2.63072L8.33556 3.57823C8.1866 3.95689 7.91748 4.27903 7.56761 4.49752L7.23937 4.68218C6.83135 4.88657 6.36254 4.94357 5.91569 4.8431L4.72675 4.51658C4.02683 4.32293 3.67688 4.22611 3.36745 4.34881C3.05801 4.47152 2.87616 4.77924 2.51243 5.39468L2.03151 6.20845C1.69057 6.78533 1.5201 7.07378 1.55319 7.38084C1.58626 7.6879 1.81448 7.93535 2.2709 8.43024L3.2755 9.52529C3.52104 9.82834 3.69536 10.3565 3.69536 10.8314C3.69536 11.3065 3.52109 11.8345 3.27553 12.1377L2.2709 13.2328C1.81448 13.7277 1.58627 13.9751 1.55319 14.2822C1.5201 14.5893 1.69057 14.8777 2.03151 15.4545L2.51242 16.2683C2.87614 16.8837 3.05801 17.1915 3.36745 17.3142C3.67688 17.4369 4.02684 17.3401 4.72677 17.1464L5.91565 16.8199C6.36258 16.7194 6.83148 16.7765 7.23954 16.9809L7.56773 17.1656C7.91754 17.3841 8.18659 17.7061 8.33553 18.0848L8.66095 19.0324C8.87489 19.6594 8.98187 19.9729 9.23651 20.1523C9.49121 20.3315 9.82951 20.3315 10.506 20.3315H11.5922C12.2688 20.3315 12.6071 20.3315 12.8617 20.1523C13.1164 19.9729 13.2234 19.6594 13.4374 19.0324L13.7628 18.0848C13.9117 17.7061 14.1807 17.3841 14.5306 17.1656L14.8587 16.9809C15.2668 16.7765 15.7357 16.7194 16.1826 16.8199L17.3715 17.1464C18.0714 17.3401 18.4214 17.4369 18.7308 17.3142C19.0403 17.1915 19.2221 16.8837 19.5858 16.2683L20.0668 15.4545C20.4077 14.8777 20.5781 14.5893 20.5451 14.2822C20.512 13.9751 20.2838 13.7277 19.8274 13.2328L18.8227 12.1377C18.5772 11.8345 18.4029 11.3065 18.4029 10.8314C18.4029 10.3565 18.5773 9.82834 18.8227 9.52529L19.8274 8.43024C20.2838 7.93535 20.512 7.6879 20.5451 7.38084C20.5781 7.07378 20.4077 6.78533 20.0668 6.20845Z"
-                stroke={pathname === "/settings/" ? "#000" : "#676767"}
-                strokeWidth="1.5"
-                strokeLinecap="round"
+        {holdIsExpanded ? (
```

Comment:
You can use isExpanded here since holdIsExpanded holds the same value as isExpanded

Reply:
fixed! thanks